src/pages/Options/Market/TradeCard/BiddingPhaseCard/BiddingPhaseCard.tsx

```diff
@@ -98,6 +101,14 @@ const BiddingPhaseCard: FC<BiddingPhaseCardProps> = ({
 	const [shortSideAmount, setShortSideAmount] = useState<OptionsTransaction['amount'] | string>('');
 	const [longPriceAmount, setLongPriceAmount] = useState<string | number>('');
 	const [shortPriceAmount, setShortPriceAmount] = useState<string | number>('');
+	const [
+		withdrawalsDisabledTooltipDismissedMarkets,
+		setWithdrawalsDisabledTooltipDismissedMarkets,
+	] = useLocalStorage(LOCAL_STORAGE_KEYS.BO_WITHDRAWALS_DISABLED_TOOLTIP_DISMISSED, []);
+
+	const withdrawalsDisabledTooltipDismissed = withdrawalsDisabledTooltipDismissedMarkets.includes(
+		optionsMarket.address
```

Comment:
optionsMarket.address is always defined?

Reply:
yep, the whole view (including the TradeCard) is blocked from rendering until we got the `optionsMarket` object from the SC.